Makefile

```diff
@@ -1,8 +1,11 @@
 PYTHON ?= python
 PIP ?= pip
 CYTHON ?= cython
+CPP_DRIVER ?= scylladb
 current_dir = $(shell pwd)
 
+export CPP_DRIVER
```

Comment:
why we do need this export?

Reply:
for setup.py to choice libname for binding
https://github.com/whiteluna/acsylla/blob/support-two-drivers/setup.py#L14-L23

Reply:
Ill will go for not having support for both drivers, so basically let´s point the cpp folder to the scylladb repository and lets hardcode the name of the library, if later on or eventually we need to have support for both of them in the code we can always implement this dual support.

Also take into account that once you introduce the usage of `cass_cluster_set_local_port_range` either you add compile time check for using or not the function or the linker will complain when you use the cassandra driver.

So TBH Ill go simple for now, just replace the cassandra dirver submodule for the syclla one and if its necessary change the name of the library used in the `setup.py` file

Reply:
Ok, no problem